packages/dnb-eufemia/src/extensions/forms/FieldBlock/FieldBlockDocs.ts

```diff
@@ -21,6 +21,16 @@ export const fieldBlockSharedProperties: PropertiesTableProps = {
     type: 'string',
     status: 'optional',
   },
+  layoutOptions: {
```

Comment:
```suggestion
  horizontalLayoutOptions: {
```

Based on the description of the `layoutOptions` prop, it seems like its only use is to set additional options for the `horizontal` layout. Perhaps the prop name should reflect so as well? 🤔